### PR TITLE
docs: editorial pass

### PR DESCRIPTION
**Pull request by alexisintech**
Resolved many Vercel comments. Checked for broken links, code references that should be in code snippets (hence, the reformatting of many tables), grammatical errors that stuck out, and updated imports in code examples.

### CODE REVIEW

File: docs/references/react/use-session-list.mdx [hunk 1]
```diff
@@ -55,33 +55,9 @@ export default function Home() {
...
-        "sessions",
-        "Holds an array of Session objects that have been registered on the client device",
-      ],
-    },
-  ]}
-/>
+| Values | Description |
+| --- | --- |
+| `isLoaded` | A boolean that is set to false until Clerk loads and initializes. |
+| `setActive()` | A function that sets the active session. In most cases, this should be used over setSession(). |
```
Comment (alexisintech) [inline docs/references/react/use-session-list.mdx:61]:
```suggestion
| `setActive()` | A function that sets the active session. In most cases, this should be used over `setSession()`. |
```

File: docs/references/react/use-session.mdx [hunk 2]
```diff
@@ -65,30 +65,9 @@ export default function Home() {
...
-    },
-    {
-      cells: ["session", "Holds the current active session for the user"],
-    },
-  ]}
-/>
+| Values | Description |
+| --- | --- |
+| `isLoaded` | A boolean that is set to false until Clerk loads and initializes. |
+| `setActive()` | A function that sets the active session. In most cases, this should be used over setSession(). |
```
Comment (alexisintech) [inline docs/references/react/use-session.mdx:71]:
```suggestion
| `setActive()` | A function that sets the active session. In most cases, this should be used over `setSession()`. |
```

File: docs/references/react/use-sign-in.mdx [hunk 3]
```diff
@@ -173,21 +173,9 @@ The `useSignIn` hook provides access to the `SignIn` object, which allows you to
...
-  },
-  {
-  cells: ["signIn", "An object that contains the current sign in attempt status and methods to create a new sign in attempt."]
-  },
-  ]}
-/>
+| Values | Description |
+| --- | --- |
+| `isLoaded` | A boolean that is set to false until Clerk loads and initializes. |
+| `setActive()` | A function that sets the active session. In most cases, this should be used over setSession(). |
```
Comment (alexisintech) [inline docs/references/react/use-sign-in.mdx:179]:
```suggestion
| `setActive()` | A function that sets the active session. In most cases, this should be used over `setSession()`. |
```

File: docs/references/react/use-sign-up.mdx [hunk 4]
```diff
@@ -178,21 +178,10 @@ The `useSignUp` hook gives you access to the `SignUp` object, which allows you t
...
-  },
-  {
-  cells: ["signUp", "An object that contains the current sign up attempt status and methods to create a new sign up attempt."]
-  },
-  ]}
-/>
+| Values | Description |
+| --- | --- |
+| `isLoaded` | A boolean that is set to false until Clerk loads and initializes. |
+| `setActive()` | A function that sets the active session. In most cases, this should be used over setSession(). |
```
Comment (alexisintech) [inline docs/references/react/use-sign-up.mdx:184]:
```suggestion
| `setActive()` | A function that sets the active session. In most cases, this should be used over `setSession()`. |
```

File: docs/quickstarts/javascript.mdx [hunk 5]
```diff
@@ -82,7 +82,7 @@ To use the ClerkJS package, you'll need to initialize the Clerk class with your
 ```js copy filename="index.js" 
 import Clerk from '@clerk/clerk-js';
 
-const clerkFrontendApi = 'pk_[publishable_key]';
+const clerkFrontendApi = `pk_${{pub_key}}`;
```
[the remark this answers is not in the document]
Comment (BRKalow) [reply docs/quickstarts/javascript.mdx:85]: It needs to be `{{pub_key}}`, but also the code block has to be wrapped in `<InjectKeys>`. I'm honestly not sure if that'll work in conjunction with `CodeBlockTabs`, but we can try!

File: docs/quickstarts/javascript.mdx [hunk 6]
```diff
@@ -91,7 +91,7 @@ await clerk.load({
 
 ```html copy filename="index.html"
 <script>
-  const clerkFrontendApi = 'pk_[publishable_key]';
+  const clerkFrontendApi = `pk_${{pub_key}}`;
```
Comment (BRKalow) [inline docs/quickstarts/javascript.mdx:94]:
```suggestion
  const clerkFrontendApi = `pk_{{pub_key}}`;
```

File: docs/references/javascript/clerk/handle-navigation.mdx [hunk 7]
```diff
@@ -42,76 +38,29 @@ Take a look at the function parameters description below for more usage details.
...
-    {
-    cells: [
-    <code>customNavigate</code>,
-    <code>(to: string) {"=>"} Promise{"<"}unknown{">"}</code>,
-    <>Allows you to define a custom navigation function</>,
-    ],
-    }
-  ]}
-/>
+  If the magic link is successfully verified on another device, there's a callback function parameter that allows custom code execution. |
```
Comment (BRKalow) [inline docs/references/javascript/clerk/handle-navigation.mdx:48]: Do multiline cells work like this? Amazing if so

File: docs/references/javascript/clerk/organization-methods.mdx [hunk 8]
```diff
@@ -61,58 +38,28 @@ The `createOrganization` method is used to create an organization programaticall
...
-  rows={[
-  {cells: [
-  <code>Promise{"<"}<a href="/docs/references/javascript/organization-membership">OrganizationMembership</a>[]{">"}</code>,
-  <>This method returns a <code>Promise</code> which resolves to the list of <code><a href="/docs/references/javascript/organization-membership">Organization</a></code>s for the current user.</>,
-  ]},
-  ]}
-/>
+| Type | Description |
+| --- | --- |
+| `Promise<OrganizationMembership[]>` | This method returns a `Promise` which resolves to the list of [`OrganizationMembership`]("/docs/references/javascript/organization-membership")'s for the current user. |
```
Comment (BRKalow) [inline docs/references/javascript/clerk/organization-methods.mdx:65]:
```suggestion
| `Promise<OrganizationMembership[]>` | This method returns a `Promise` which resolves to the list of [`OrganizationMembership`](/docs/references/javascript/organization-membership)'s for the current user. |
```

File: docs/references/javascript/clerk/sign-in.mdx [hunk 9]
```diff
@@ -75,26 +70,10 @@ Render the `<SignIn />` component to an HTML `<div>` element.
...
-  <a href="#sign-in-props"><code>SignInProps</code></a>,
-  "The properties to pass to the `<SignIn />` component",
-  ],
-  },
-  ]}
-/>
+| Name | Type | Description |
+| --- | --- | --- |
+| `node` | [`HTMLDivElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDivElement) | The container `<div>` element used to render in the `<SignIn />` component |
+| `props?` | [`SignInProps`]("#sign-in-props") | The properties to pass to the `<SignIn />` component |
```
Comment (BRKalow) [inline docs/references/javascript/clerk/sign-in.mdx:76]:
```suggestion
| `props?` | [`SignInProps`](#sign-in-props) | The properties to pass to the `<SignIn />` component |

```

File: docs/references/javascript/clerk/sign-in.mdx [hunk 10]
```diff
@@ -217,19 +186,9 @@ Opens the `<SignIn />` component as an overlay at the root of your HTML `body` e
...
-  "props?",
-  <a href="#sign-in-props"><code>SignInProps</code></a>,
-  "The properties to pass to the `<SignIn />` component",
-  ],
-  },
-  ]}
-/>
+| Name | Type | Desciption |
+| --- | --- | --- |
+| `props?` | [`SignInProps`]("#sign-in-props") | The properties to pass to the `<SignIn />` component |
```
Comment (BRKalow) [inline docs/references/javascript/clerk/sign-in.mdx:191]:
```suggestion
| `props?` | [`SignInProps`](#sign-in-props) | The properties to pass to the `<SignIn />` component |

```

File: docs/references/javascript/clerk/user-button.mdx [hunk 11]
```diff
@@ -152,97 +132,23 @@ Unmount and run cleanup on an existing `<UserButton />` component instance.
...
-  "node",
-  <a href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLDivElement"><code>HTMLDivElement</code></a>,
-  <>The container <code>{`<div>`}</code> element with a rendered {"<UserButton />"} component instance</>
-  ],
-  },
-  ]}
-/>
+| Name | Type | Description |
+| --- | --- | --- |
+| `node` | [`HTMLDivElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDivElement) | The container `<div>` element with a rendered {"<UserButton />"} component instance. |
```
Comment (BRKalow) [inline docs/references/javascript/clerk/user-button.mdx:137]:
```suggestion
| `node` | [`HTMLDivElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDivElement) | The container `<div>` element with a rendered `<UserButton />` component instance. |
```

File: docs/references/javascript/clerk/handle-navigation.mdx (hunk 7 above)
Comment (alexisintech) [reply docs/references/javascript/clerk/handle-navigation.mdx:48]: yup :~)

styling needs some work though. text in tables have weird line height anyways, and then li's have normal line-height i'm assuming.

Screenshot 2023-08-17 at 11 16 57

File: docs/references/javascript/email-address/email-address.mdx [hunk 12]
```diff
@@ -13,7 +13,7 @@ The EmailAddress object is a model around an email address. Email addresses are
 
 Email addresses must be verified to ensure that they can be assigned to their rightful owners. The EmailAddress object holds all necessary state around the verification process.
 
-The verification process always starts with the EmailAddress.prepareVerification() method, which will send a one-time verification code via an email message. The second and final step involves an attempt to complete the verification by calling the EmailAddress.attemptVerification() method, passing the one-time code as a parameter.
+The verification process always starts with the [EmailAddress.prepareVerification()](,/docs/references/javascript/email-address/verification#prepare-verification) method, which will send a one-time verification code via an email message. The second and final step involves an attempt to complete the verification by calling the [EmailAddress.attemptVerification()](/docs/references/javascript/email-address/verification#attempt-verification) method, passing the one-time code as a parameter.
```
Comment (BRKalow) [inline docs/references/javascript/email-address/email-address.mdx:16]:
```suggestion
The verification process always starts with the [EmailAddress.prepareVerification()](/docs/references/javascript/email-address/verification#prepare-verification) method, which will send a one-time verification code via an email message. The second and final step involves an attempt to complete the verification by calling the [EmailAddress.attemptVerification()](/docs/references/javascript/email-address/verification#attempt-verification) method, passing the one-time code as a parameter.

```

**Review by BRKalow — COMMENTED** [review]
⭐ 👏

**Review by BRKalow — APPROVED** [review]
🚀

File: docs/references/javascript/clerk/session-methods.mdx [hunk 13]
```diff
@@ -50,16 +50,16 @@ function setActive({
...
 > This method will eventually replace `setSession`.
 
 ### `SetActiveParams`
 
 | Name | Type | Description |
 | --- | --- | --- |
-| `session` | `Session` \| `string` \| `null` | The session resource or session id (string version) to be set as active. If `null`, the current session is deleted. |
-| `organization` | [`Organization`](/docs/references/javascript/organization/organization) \| `string` \| `null` | The organization resource or organization id (string version) to be set as active in the current session. If `null`, the currently active organization is removed as active. |
+| `session` | `Session \| string \| null` | The session resource or session ID (string version) to be set as active. If `null`, the current session is deleted. |
+| `organization` | `[Organization](/docs/references/javascript/organization/organization) \| string \| null` | The organization resource or organization ID (string version) to be set as active in the current session. If `null`, the currently active organization is removed as active. |
```
Comment (BRKalow) [inline docs/references/javascript/clerk/session-methods.mdx:62]: The prior syntax was correct, this won't render the link properly as-is.

Comment (alexisintech) [reply docs/references/javascript/clerk/session-methods.mdx:62]: ACK. its not gonna be consistent with how other types are formatted :~(

**Review by BRKalow — APPROVED** [review]

**Pull request merged**